NEWS.md

```diff
@@ -7,6 +7,8 @@
 * `get_source_expressions()` can handle Sweave/Rmarkdown documents with reference chunks like `<<ref_file>>` (#779, @MichaelChirico).
   Note that these are simply skipped, rather than attempting to retrieve the reference and also lint it.
 
+* `spaces_inside_linter()` produces lints for spaces inside `[[` (#1673, @IndrajeetPatil).
```

Comment:
I think we put this under `Changes to defaults` or `New and improved features`, it's not really a "bug" in the sense that we usually use it for {lintr}